core/src/epicli/data/common/defaults/configuration/postgresql.yml

```diff
@@ -40,14 +40,6 @@ specification:
               - name: wal_level
                 value: replica
                 when: replication
-          - name: Archiving
```

Comment:
How about keeping these settings but in accordance with Repmgr docs?
archive_mode

> We suggest setting archive_mode to on (and archive_command to /bin/true; see below) even if you are currently not planning to use WAL file archiving.
> 
> This will make it simpler to set up WAL file archiving if it is ever required, as changes to archive_mode require a full PostgreSQL server restart, while archive_command changes can be applied via a normal configuration reload.
> 
> However, repmgr itself does not require WAL file archiving.

Source: https://repmgr.org/docs/repmgr.html#CONFIGURATION-POSTGRESQL